pip/commands/show.py

```diff
@@ -85,13 +86,11 @@ def search_packages_info(query):
             entry_points = dist.get_metadata_lines('entry_points.txt')
             package['entry_points'] = entry_points
 
-        installer = None
         if dist.has_metadata('INSTALLER'):
             for line in dist.get_metadata_lines('INSTALLER'):
                 if line.strip():
-                    installer = line.strip()
+                    package['installer'] = line.strip()
                     break
```

Comment:
This change is not just cosmetic. When `pip show` prints fields of `package` it does the following `package.get('field', '')`.  If `package[field] == None` then `None` gets printed. This seemed to be the only case where a field was being assigned `None` rather than being undefined. (admittedly its kind of brittle)